### PR TITLE
Bubble up daemon exit message to zli

### PR DESCRIPTION
**Pull request by sidpremkumar**
## Description of the change

Currently we do not bubble up the ExitMessage that the daemon receives to `zli tunnel`. @tflpd correctly pointed out that this would be a common debug step for most customers, so this PR adds that functionality. 
```
> zli tunnel
The Kube Deamon has gotten an exit message from Bastion. Please try logging in again and restarting the daemon.
Exit Message: closing message received; websocket closed
```

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] Refactor (code change with no functionality change)
- [ ] Enhancement (minor change below the level of a feature)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Remove feature (removes a feature we don't support anymore)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related JIRA tickets

Relates to JIRA: CWC-1260

## Relevant release note information

Release Notes:

## Potential Security Impacts

Does this PR have any security impact?
Does it fix a security issue?
Does it add attack surface?
Why do we think this change is safe?

### Development

- [ ] Does the code changed/added as part of this pull request have test coverage?
- [ ] Do all tests related to the changed code pass in development?
- [ ] Have you ensured that at least one other person has tested your code?
- [ ] Have you tested the code?
- [ ] Have you attached any pictures (if relevant)?

### CODE REVIEW

**Review by tflpd — APPROVED** [review]

**Comment by sidpremkumar** [comment]
Tested again by manually turned off my agent: 
2021-12-10 at 4 03 PM

File: kube-go-assets/bctl/daemon/httpserver/httpserver.go [hunk 1]
```diff
@@ -28,6 +29,10 @@ const (
 	getChallenge  = false
 )
 
+type StatusMessage struct {
```
Comment (sidpremkumar) [inline kube-go-assets/bctl/daemon/httpserver/httpserver.go:32]: @lgmugnier not sure if this is the best place for this message

**Comment by sidpremkumar** [comment]
Tested again by manually turned off my agent: 
2021-12-10 at 4 03 PM

File: kube-go-assets/bctl/daemon/httpserver/httpserver.go (hunk 1 above)
Comment (lgmugnier) [reply kube-go-assets/bctl/daemon/httpserver/httpserver.go:32]: for now I think it's fine

**Review by lgmugnier — APPROVED** [review]

**Pull request merged**